### PR TITLE
fix: Only look for blueprints on generate from our blueprint directory.

### PR DESCRIPTION
**Pull request by hansl**
Fixes https://github.com/angular/angular-cli/issues/266

### CODE REVIEW

File: addon/ng2/commands/generate.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as EmberGenerateCommand from 'ember-cli/lib/commands/generate';
+import * as path from 'path';
+
+
+const GenerateCommand = EmberGenerateCommand.extend({
+  name: 'generate',
+
+  lookupBlueprint: function(name) {
+    return Blueprint.lookup(name, {
```
Comment (filipesilva) [inline addon/ng2/commands/generate.ts]: Missing import of Blueprint, I believe.

Comment (hansl) [inline addon/ng2/commands/generate.ts]: Done.

**Comment by filipesilva** [comment]
This PR will block all blueprints that are not in this repo, which does block unsupported blueprints from `ember-cli` but will also block blueprints from `angular-cli` addons.

**Comment by hansl** [comment]
It's okay for now, when we release we'll have to revisit blueprints.

**Pull request merged**